src/Parallel/Main.hpp

```diff
@@ -97,9 +98,22 @@ Main<Metavariables>::Main(CkArgMsg* msg) noexcept
   namespace bpo = boost::program_options;
   try {
     bpo::options_description command_line_options;
-    command_line_options.add_options()
-        ("help,h", "Describe program options")
-        ("check-options", "Check input file options")
+    command_line_options.add_options()                 //
+        ("help,h", "Describe program options")         //
+        ("check-options", "Check input file options")  //
+        ("dump-source-tree-as", bpo::value<std::string>(),
+         "If specified then an gzip archive of the source tree is dumped "
+         "with the specified and the executable is terminated. The archive can "
+         "be expanded using 'tar -xzf ARCHIVE.tar.gz'")  //
+        ("dump-paths",
+         "Dump the PATH, CPATH, LD_LIBRARY_PATH, LIBRARY_PATH, and "
+         "CMAKE_PREFIX_PATH at compile time.")  //
+        ("dump-environment",
+         "Dump the result of printenv at compile time.")  //
+        ("dump-library-versions",
+         "Dump the contents of SpECTRE's LibraryVersions.txt")  //
+        ("dump-only",
+         "Exit after dumping requested information.")  //
```

Comment:
I used these to get clang-format to keep this formatting instead of combining things randomly. But as I type this I realize I can just disable clang-format for this section.